tests/test_spark.py

```diff
@@ -72,6 +72,29 @@ def test_mesos_label_support():
     assert expected in driver_task_info['labels']
 
 
+@pytest.mark.sanity
+def test_dispatcher_liveliness_after_malformed_request():
+    # Get dispatcher ip and port
+    dispatcher_details = sdk_cmd._get_task_info(utils.SPARK_PACKAGE_NAME)
+    ip = dispatcher_details['statuses'][0]['container_status']['network_infos'][0]['ip_addresses'][0]['ip_address']
+    port = dispatcher_details['discovery']['ports']['ports'][0]['number']
+
+    # Sends resource/<filename> as a submission request to dispatcher and returns action field
+    def submit_dispatcher_request(request_filename):
+        with open(os.path.join(THIS_DIR, 'resources', request_filename), 'r') as file:
+            request = file.read()
+        curl_command = '''curl -d '{}' -H "Content-Type: application/json" -X POST "http://{}:{}/v1/submissions/create"''' \
```

Comment:
let's use HTTPS here with a corresponding change in `curl` command: it will need `--insecure`